english/love/love.md

```diff
@@ -10,7 +10,7 @@ There is no use moping around wondering the ifs, buts, could've, would've,
 and should'ves. Time goes on and drags you with it. Whether you are in or
 out of love, make the best of now while being aware of the worst of before.
 
-Of course, there is no substitute for the experince of it. It must be
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.